godtools/GeneralExtensions/UIFont.swift

```diff
@@ -77,4 +85,27 @@ extension UIFont {
         return UIFont(name: fontName, size: fontSize)!
     }
     
+    
+    func setBold() -> UIFont {
+        if isBold {
+            return self
+        } else {
+            let fontDescriptorVar: UIFontDescriptor = fontDescriptor.withSymbolicTraits(.traitBold)!
+            return UIFont(descriptor: fontDescriptorVar, size: 0)
+        }
+    }
+    
+    func setItalic()-> UIFont {
+        if(isItalic) {
+            return self
+        } else {
```

Comment:
Since you return from the if, there's no need for the else statement and block. Just remove the else and there's one less level of indentation.